lib/utils/yaml-utils.ts

```diff
@@ -2,8 +2,9 @@ import * as fs from 'fs';
 import * as yaml from 'js-yaml';
 import * as eks from '@aws-cdk/aws-eks';
 import request from 'sync-request';
+import { KubernetesManifest } from '@aws-cdk/aws-eks';
 
-export function readYamlFromDir(dir: string, cluster: eks.Cluster): void {
+export function readYamlFromDir(dir: string, cluster: eks.Cluster, namespaceManifest: KubernetesManifest): void {
```

Comment:
we should rename this function to `applyYamlFromDir`

Reply:
changed.